File: solutions/java/001.java
class Problem001 {
    public int solution() {
        int sum = 0;

        for (int n = 0; n < 1000; n++) {
            if (n % 3 == 0 || n % 5 == 0) {
                sum += n;
            }
        }

        return sum;
    }

    public static void main(String[] args) {
        Problem001 problem = new Problem001();
        System.out.println(problem.solution());
    }
}

File: solutions/javascript/001.js
function solution() {
    var sum = 0;

    for (var i = 0; i < 1000; i++) {
        if (i % 5 === 0 || i % 3 === 0) {
            sum += i;
        }
    }

    return sum;
}

if (require.main === module) {
    console.log(solution());
}
